ajax/libs/video.js-chromecast/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "video.js-chromecast",
+  "version": "2.0.8",
+  "filename": "videojs-chromecast.min.js",
+  "description": "Chromecast plugin for videojs",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/benjipott/video.js-chromecast.git"
+    },
```

Comment:
Let's modify from `    },` to `  },`. Just need to delete two space to fit the format and pass the CI.